foxpuppet/windows/browser/window.py

```diff
@@ -53,5 +89,6 @@ def open_window(self, private=False):
                     *self._file_menu_new_window_button_locator).click()
 
         return self.wait.until(
-            new_browser_window_is_opened(self.selenium, handles_before),
+            expected.new_browser_window_is_opened(
+                self.selenium, handles_before),
```

Comment:
If not done yet please file an issue so we can discuss to make that easier to use. Having to push the former handles as parameter doesn't make that much sense to me. It should be all transparent in the method itself.